backend/db/migrations/20220715162810-create-booking.js

```diff
@@ -43,10 +43,6 @@ module.exports = {
         allowNull: false,
         type: Sequelize.DATE
       },
-      stripePaymentIntentId: {
-        allowNull: true,
-        type: Sequelize.STRING
-      },
```

Comment:
I believe this field was added on your `create booking with Stripe and Stripe webhooks implemented` branch(PR82). Just to make sure, are we deleting this field because we refactored our API for the webhook? Or was it a git issue that deleted it depending on where you created this branch?

Reply:
Hi Jefferson, great catch, I totally forgot to mention it, the stripePaymentIntentId column was added when I initially started on implementing Stripe API, but once I finished the feature, I realized this column is no longer needed, therefore, I removed it.